leaf-ui/scripts/main.js

```diff
@@ -26,7 +25,10 @@ var reader;
 
 //Properties for both core and simulator.
 //TODO: merge this two arrays in order to make use the same name for all
-var satvalues = {satisfied: 2, partiallysatisfied: 1, partiallydenied: -1, denied: -2, unknown: 4, conflict: 3, none: 0};
+var satvalues = {
```

Comment:
Why are we still using the iStar values here. How are the four conflict types stored?